jest.config.json

```diff
@@ -0,0 +1,4 @@
+{
+	"preset": "ts-jest",
+	"testEnvironment": "node"
+}
```

Comment:
Did you purposefully omit `coverageProvider`?

Reply:
Yeah, it didn't seem important at this point, we can add it later though if needed.